manifest.production.yml

```diff
@@ -3,7 +3,7 @@ buildpack: python2_buildpack
 health-check-type: port
 instances: 8
 memory: 1G
-disk_quota: 512M
+disk_quota: 2G
```

Comment:
The waitress server uses `/tmp` to buffer large outputs, so even though the download code is now not hogging memory, the HTTP response still needs to sit in full somewhere in our app, and disk is probably better than RAM.